docs/dev.rst

```diff
@@ -1,20 +1,570 @@
+
+    Example: ``namespace_inject: hed`` and ``unmanaged: True``
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - zip_url: https://github.com/SalesforceFoundation/EDA/archive/master.zip
+              subfolder: EDA-master/dev_config/src/admin_config
+              namespace_inject: hed
+              unmanaged: True
+
+..
+
+    The namespace tokens are replaced with an empty string instead of the namespace, effectively stripping the tokens from the files and filenames.
+
+
+
+Strip Namespace Prefixes
+************************
+
+If the metadata in the zip to be deployed has references to a namespace prefix, use the ``namespace_strip`` option to remove them.
+
+    Example: ``namespace_strip: npsp``
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - zip_url: https://github.com/SalesforceFoundation/CumulusReports/archive/master.zip
+              subfolder: CumulusReports-master/src
+              namespace_strip: npsp
+
+..
+
+    When ``update_dependencies`` runs, the zip is retrieved and the string ``npsp__`` is stripped from all files and filenames in the zip before deployment.  This option is most useful when setting up an unmanaged development environment for an extension package that normally uses managed dependencies.
+    
+    This example takes the NPSP Reports & Dashboards project's unmanaged metadata and strips the references to ``npsp__`` to deploy it against an unmanaged version of NPSP.
+
+
+GitHub Repository Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+GitHub repository dependencies create a dynamic dependency between the current project and another project on GitHub that uses CumulusCI to manage its dependencies.
+
+    Example: Salesforce EDA
+
+.. code-block:: yaml
+ 
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+
+When ``update_dependencies`` runs, these steps are taken against the referenced repository.
+
+* Look for the ``cumulusci.yml`` file and parse if found.
+* Determine if the project has subfolders under unpackaged/pre.  If found, deploys them first.
+* Determine if the project specifies any dependencies in the ``cumulusci.yml`` file.  If found, deploys them next in the queue.
+* Determine if the project has a namespace configured in the ``cumulusci.yml`` file. If found, treats the project as a managed package unless the unmanaged option is also ``True``.
+* If the project has a namespace and is not set for unmanaged, use the GitHub API to get the latest release and install it.
+* If the project is an unmanaged dependency, the ``src`` or ``force-app`` directory is deployed.
+* Determine if the project has subfolders under unpackaged/post. If found, deploys them next. Namespace tokens are replaced with ``namespace__`` or an empty string, depending on if the dependency is considered managed or unmanaged.
```

Comment:
```suggestion
* Determine if the project has subfolders under ``unpackaged/post``. If found, deploy them next. Namespace tokens are replaced with ``namespace__`` if the project is being installed as a managed package, or an empty string otherwise.
```